groups/sig-etcd/groups.yaml

```diff
@@ -0,0 +1,15 @@
+groups:
+  - email-id: etcd-security@kubernetes.io
+    name: etcd-security
```

Comment:
I need to keep using etcd-security, otherwise the test case `TestMergedGroupsConfig` will fail,

```

--- FAIL: TestMergedGroupsConfig (0.00s)
    groups_test.go:110: 1 duplicate groups: security
```